app/src/main/java/org/mozilla/fenix/migration/MigrationProgressActivity.kt

```diff
@@ -0,0 +1,151 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.migration
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.activity_migration.*
+import kotlinx.android.synthetic.main.migration_list_item.view.*
+import mozilla.components.support.migration.AbstractMigrationProgressActivity
+import mozilla.components.support.migration.Migration
+import mozilla.components.support.migration.Migration.Bookmarks
+import mozilla.components.support.migration.Migration.History
+import mozilla.components.support.migration.Migration.Logins
+import mozilla.components.support.migration.Migration.Settings
+import mozilla.components.support.migration.MigrationResults
+import mozilla.components.support.migration.state.MigrationProgress
+import mozilla.components.support.migration.state.MigrationStore
+import org.mozilla.fenix.HomeActivity
+import org.mozilla.fenix.IntentReceiverActivity
+import org.mozilla.fenix.R
+import org.mozilla.fenix.ext.components
+
+class MigrationProgressActivity : AbstractMigrationProgressActivity() {
+    private val statusAdapter = MigrationStatusAdapter()
+    override val store: MigrationStore by lazy { components.migrationStore }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        setContentView(R.layout.activity_migration)
+        init()
+        super.onCreate(savedInstanceState)
```

Comment:
nit: Is calling `super.onCreate()` at the end on purpose? I have no problem with that, it just looks different than usual. :)

Reply:
It's funny, I was reading [this](https://stackoverflow.com/a/18874519) to see if it mattered for onStart and onStop, but forgot about onCreate. :)

I'll put it on the top!